src/globals.h

```diff
@@ -549,7 +549,14 @@ EXTERN int	garbage_collect_at_exit INIT(= FALSE);
 #define t_typealias		(static_types[90])
 #define t_const_typealias	(static_types[91])
 
-EXTERN type_T static_types[92]
+#define t_tuple_any		(static_types[92])
+#define t_const_tuple_any	(static_types[93])
+
+#define t_tuple_empty		(static_types[94])
+#define t_const_tuple_empty	(static_types[95])
+
+
+EXTERN type_T static_types[114]
```

Comment:
```suggestion
EXTERN type_T static_types[96]
```

Reply:
In the earlier PRs, there were more tuple static types.  I removed them as they are no longer needed.  I forgot to update this one.